solutions/swb-reference/integration-tests/tests/isolated/sshKeys/create.test.ts

```diff
@@ -13,34 +13,37 @@ describe('cannot create SSH key', () => {
   const setup: Setup = new Setup();
   let adminSession: ClientSession;
   let project: { id: string };
+  let costCenterId: string;
   const randomTextGenerator = new RandomTextGenerator(setup.getSettings().get('runId'));
 
+  beforeAll(async () => {
+    adminSession = await setup.getDefaultAdminSession();
+  });
+
+  afterAll(async () => {
+    await setup.cleanup();
+  });
+
   beforeEach(async () => {
     expect.hasAssertions();
-    adminSession = await setup.getDefaultAdminSession();
     const { data: costCenter } = await adminSession.resources.costCenters.create({
       name: randomTextGenerator.getFakeText('fakeCostCenterName'),
       accountId: setup.getSettings().get('defaultHostingAccountId'),
       description: 'a test object'
     });
-
-    const { data } = await adminSession.resources.projects.create({
-      name: randomTextGenerator.getFakeText('fakeProjectName'),
-      description: 'Project for list users for project tests',
-      costCenterId: costCenter.id
-    });
-
-    project = data;
-  });
-
-  afterEach(async () => {
-    await setup.cleanup();
+    costCenterId = costCenter.id;
   });
 
   describe('when the user already has a key for that project', () => {
     let existingSshKeyId: string;
 
     beforeEach(async () => {
+      const { data } = await adminSession.resources.projects.create({
```

Comment:
Nitpick: Since there's only one test in the `describe` block, I would just put the `beforeEach` code inside of the `it throws 400 error` test.

If there were multiple tests later on, and the developer thinks each test needs the same setup, they can move that code to `beforeEach`.

Reply:
I see what you mean. I think this test got refactored in the PaaB branch anyway so I won't change anything here.